notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "specviz.load_data(spec_url)  #<- auto-downloads from the internet\n",
+    "\n",
+    "# downalod to a local file and load it by name\n",
+    "!wget --content-disposition $spec_url\n",
+    "specviz.load_data('spec-1323-52797-0012.fits') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    Internally this should do the same as loading the `Spectrum1D` - i.e. first convert it to `Spectrum1D` using `read` and then load it into glue/jdaviz. That way the user can know exactly what is happening when they do it with a GUI button or at the command line."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, the above should be possible using the \"open\" or \"import data\" button - the intention would be for it to all behave just as described above, but via a file browser (or command-line file path) instead of via a notebook method call."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Multiple spectra in one `specviz` "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because glupyter/glue support multiple data being loaded at the same time, it should also be possible to load data even if it's not all getting displayed initially.\n",
+    "\n",
+    "1.  In the notebook this is just a different method that otherwise behaves the same as `load_data`, just doesn't replace what's being shown - E.g.,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.add_data('some_other_spectrum.fits') #<- this would do exactly the same as `load_data` but just not show it\n",
+    "# OR \n",
+    "specviz.add_data('some_other_spectrum.fits', show=True)  #<- this would automatically add the flux as a new layer in the UI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, if the user does \"import data\" a second (or additional) time, `add_data` as described above can be used instead of `load_data`, as the user can always simply go into the viewer and change what's being shown via check boxes/drag-and-drop."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Uncertainties/mask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the `Spectrum1D` object loaded contains an uncertainty and/or mask attribute, these should be loaded ointo glue when `load_data` is called.  However, by default only the spectrum itself should be *shown* at that time. The user can show it later using standard `jdaviz` interactions if desired. We *may* wish some syntactic convenience for the notebook as well - e.g.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.show_uncertainty()\n",
+    "specviz.show_mask()"
+   ]
```

Comment:
Uncertainties could be shown as errorbars.
Not sure about how to graphically represent a spectrum mask.

Reply:
Good point, I was leaving this open, but I like @havok2063's suggestions. for the uncertainty.  For the mask that might work, but an alternative inspired by splot is to have an "X" symbol at the location where the point *would* have been if it weren't masked.  That has the advantage that you can have different masks for different spectra and use the same color-scheme (which is impossible) otherwise. 

I'll put in the notebook that as a starting point, but note that the error bar and red column ideas *also*, saying we want to leave them open as options for user testing, etc.